.github/workflows/owasp-gradle-scan.yml

```diff
@@ -38,6 +38,8 @@ jobs:
         uses: gradle/actions/setup-gradle@v4
         with:
           gradle-version: wrapper
+          gradle-home-cache-includes: |
+            dependency-check-data
```

Comment:
issue: this parameter has a default value. In order to keep the old behavior and add the new folder we need to specify folders from the default value as well

```yaml
  gradle-home-cache-includes:
    description: Paths within Gradle User Home to cache.
    required: false
    default: |
        caches
        notifications
```